Models/DTOs/CommentInputDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿using AonFreelancing.Attributes;
+using System.ComponentModel.DataAnnotations;
+
+namespace AonFreelancing.Models.DTOs
+{
+    public class CommentInputDTO
+    {
+        [Required(ErrorMessage ="Can't create empty comment !")]
+        public string Content { get; set; }
+
+        [MaxFileSize(1024 * 1024 * 5)]
+        [AllowedFileExtensions([".jpg", ".jpeg", ".png"])]
```

Comment:
Create a list for file extensions in constants file, to use it for all other files and MaxFileSize.

Reply:
const only works with primitive types or immutable types so i will use : static readonly , does this true ?

Reply:
I can't use const for array or list either for readonly because it need changes on the service.
To be discussed.

Reply:
It's up to if you want to create a string const variables for all file extensions and pass it to the attribute.

Reply:
solved